nevergrad/benchmark/xpbase.py

```diff
@@ -139,22 +116,20 @@ class Experiment:
     """
 
     # pylint: disable=too-many-arguments
-    def __init__(self, function: instru.InstrumentedFunction,
-                 optimizer: Union[str, base.OptimizerFamily], budget: int, num_workers: int = 1,
+    def __init__(self, function: fbase.ExperimentFunction,
+                 optimizer: Union[str, obase.OptimizerFamily], budget: int, num_workers: int = 1,
                  batch_mode: bool = True, seed: Optional[int] = None,
                  cheap_constraint_checker: Optional[Callable[[Any], Any]] = None,
                  ) -> None:
-        assert isinstance(function, instru.InstrumentedFunction), ("All experiment functions should derive from InstrumentedFunction")
+        assert isinstance(function, fbase.ExperimentFunction), ("All experiment functions should "
+                                                                "derive from ng.functions.ExperimentFunction")
         self.function = function
         # Conjecture on the noise level.
-        if not self.function.instrumentation.probably_noisy:
-            if hasattr(self.function, "_parameters") and "noise_level" in self.function._parameters and self.function._parameters["noise_level"] > 0:  # type: ignore
```

Comment:
@teytaud the `probably_noisy` tag is now dealt with directly in `ArtificialFunction`, which is safer